packages/mui-types/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/mui/material-ui/tree/master/packages/mui-types",
   "scripts": {
-    "build": "mkdir build && cpy index.d.ts build/ && yarn build:copy-files",
+    "build": "mkdir build && cpy index.d.ts build/ && cpy OverridableComponentAugmentation.ts build/ && yarn build:copy-files",
```

Comment:
Locally this copies the `OverridableComponentAugmentation.ts` file in the `/build` folder. But when trying to install `https://pkg.csb.dev/mui/material-ui/commit/c5761e07/@mui/types` it doesn't contain this file (this is the commit that altered the build script - https://github.com/mui/material-ui/pull/32735/commits/c5761e079b6b1a79e8e91ce60a38df43b0a90260). cc @michaldudak maybe you will have an idea of what is happening 😕

Reply:
The `files` field of this package.json has just `[ "index.d.ts" ]`. Perhaps this causes the problem by not including other files in the package.

Reply:
Yep, listing the new file fixed the issue. Thanks 🙏